src/app/app.component.html

```diff
@@ -1329,6 +1331,13 @@ <h3 (click)="optsVisible = !optsVisible" style="cursor: pointer">
             <option value="below">Below</option></select
           ><br />
         </div>
+        <div *ngIf="chart.options.includes('dataLabelPosition')">
```

Comment:
The dropdown for data label position on the demo should be placed after the show data label checkbox for better user experience. Currently it's placed after legend position. 
![Screenshot 2024-01-09 at 5 53 40 PM](https://github.com/swimlane/ngx-charts/assets/47932418/e1c4fe23-2222-4640-984a-46c27ad3bea1)